.github/workflows/ci.yml

```diff
@@ -1,7 +1,13 @@
 name: doctest-CI
+
 on:
-  - push
-  - pull_request
+  push:
+    branches:
+      - main
```

Comment:
For Agda, we also have dedicated branches `ci-*` that run the CI.  Useful if you want feedback before or without opening a PR.

E.g. https://github.com/agda/agda/blob/dc56491ca9399ad4f55341e693308b44f354770c/src/github/workflows/stack.yml#L3-L20
(note that the `&`-YAML-syntax is not understood by github actions, so we expand it in a normalization step when constructing the actual `.github/workflows/stack.yml`).

Reply:
Ah, that sounds good.  I was using "drapft" PRs for the very same purpose.